osu.Game.Rulesets.Osu/Difficulty/OsuHitObjectDifficulty.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Osu.Difficulty
+{
+    public class OsuHitObjectDifficulty
+    {
```

Comment:
On this project, is it considered a good practice to define public fields instead of properties?

Reply:
I would prefer properties, myself.  Having said that, this class should be a struct anyway.

Reply:
This is actually part of a chart implementation, which probably doesn't belong in this PR so I can just delete it from this branch. Little style comments are welcome though, it's my first time with C#, I usually work with C++.